### PR TITLE
Remove next call

### PR DESCRIPTION
**Pull request by foundingnimo**
Align with new HTTP handling

### CODE REVIEW

File: src/index.ts [hunk 1]
```diff
@@ -140,9 +140,10 @@ export class SmartsheetConnector extends CoreConnector {
         ).catch(console.error)
         res.send('')
       }
+      return true
+    } else {
```
Comment (lagache) [inline src/index.ts:144]: else isn't necessary

Comment (lagache) [reply src/index.ts:144]: just being picky here

Comment (foundingnimo) [reply src/index.ts:144]: It is true that it isn't necessary- I just like this code type personally but I don't mind either way here :)

File: package.json [hunk 2]
```diff
@@ -1,7 +1,7 @@
 {
   "name": "reshuffle-smartsheet-connector",
-  "version": "0.0.1",
-  "description": "Reshuffle connectors for Smartsheet",
+  "version": "0.0.2",
```
Comment (avnerbraverman) [inline package.json]: Should be changed by @ashevat through npm

**Review by avnerbraverman — CHANGES_REQUESTED** [review]

File: package.json (hunk 2 above)
Comment (foundingnimo) [inline package.json]:
```suggestion
  "version": "0.0.1",
```

**Review by avnerbraverman — APPROVED** [review]

**Pull request merged**